src/Controller/Bancontact.php

```diff
@@ -158,4 +158,40 @@ public function process_standard_payment($order, $amount, $customer_id)
 
 	}
 
+	/**
+	 * @return bool|void
+	 */
+	public function process_admin_options() {
```

Comment:
is this in the task scope?

Reply:
PM asked for it in order to have all payment methods having the same behaviour